model/src/payouts.rs

```diff
@@ -86,6 +86,15 @@ impl Payouts {
             fee,
         )?;
 
+        // Overwrite the short liquidation upper bound with the maximum price that Olivia can attest
+        // to
+        {
+            let n_payouts = payouts.len() - 1;
```

Comment:
I didn't even think about it haha.

I prefer this because otherwise we rely on knowing what the implementation of `payouts::inverse::calculate` does to ensure that we choose the very last payout (which is crucial!).